google/auth/_refresh_worker.py

```diff
@@ -75,7 +75,7 @@ def __getstate__(self):
 
     def __setstate__(self, state):
         """Pickle helper that deserializes the _lock attribute."""
-        state["_key"] = threading.Lock()
+        state["_lock"] = threading.Lock()
```

Comment:
is there a test we could add?

Reply:
ACK - Going to wait to hear back on https://github.com/googleapis/google-auth-library-python/issues/1478 before doing more